{"uuid": "1f680f4f-fea9-4e7a-b425-8d76e7f30f24", "children": ["fafb7046-2140-4d8b-8df8-c2a931f530d3"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "UnboundLocalError: cannot access local variable 'driver' where it is not associated with a value\n", "trace": "  File \"/Users/oleksandrkozlov/PycharmProjects/test_selenium/.venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/oleksandrkozlov/PycharmProjects/test_selenium/.venv/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Users/oleksandrkozlov/PycharmProjects/test_selenium/.venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/oleksandrkozlov/PycharmProjects/test_selenium/.venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/oleksandrkozlov/PycharmProjects/test_selenium/.venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/oleksandrkozlov/PycharmProjects/test_selenium/test/fixtures/config_fixtures.py\", line 37, in driver\n    driver.quit()\n    ^^^^^^\n"}, "start": 1743768870690, "stop": 1743768870690}], "start": 1743768870690, "stop": 1743768870706}